python/tests/unit/distributed/dataset_input_metadata_translator_test.py

```
@@ -242,16 +244,17 @@ def test_translator_correctness(self, _, mocked_dataset_info: MockedDatasetInfo)
                 is_heterogeneous=graph_metadata_pb_wrapper.is_heterogeneous,
                 expected_entity_types=graph_metadata_pb_wrapper.edge_types,
             )
-            serialized_positive_label_info_iterable: list[
-                Optional[SerializedTFRecordInfo]
-            ]
+            serialized_positive_label_info_iterable: list[SerializedTFRecordInfo]
             if isinstance(
                 serialized_graph_metadata.positive_label_entity_info, abc.Mapping
             ):
                 serialized_positive_label_info_iterable = list(
                     serialized_graph_metadata.positive_label_entity_info.values()
                 )
-            else:
+            elif isinstance(
+                serialized_graph_metadata.positive_label_entity_info,
+                SerializedTFRecordInfo,
+            ):
```

should we raise an error if this is also false?